src/sql/platform/common/pathUtilities.ts

```diff
@@ -39,11 +39,13 @@ export function resolveFilePath(uri: string, filePath: string, rootPath?: string
 }
 
 export function getRootPath(contextService: IWorkspaceContextService): string | undefined {
-	let isWorkspace = contextService.getWorkbenchState() === WorkbenchState.WORKSPACE;
-	if (isWorkspace) {
-		let folder = contextService.getWorkspace().folders[0];
-		if (folder && folder.uri) {
-			return folder.uri.fsPath;
+	if (contextService) {
```

Comment:
1. If you do this make sure you update the parameter as well `IWorkspaceContextService | undefined`
2. I would suggest not changing this and instead passing in a mock context service in your tests anyways